zabbix/defaults.yaml

```diff
@@ -24,7 +24,7 @@ zabbix:
       - /etc/zabbix/zabbix_agentd.d/
   server:
     pkgs:
-      - zabbix-server-mysql
+      - zabbix-server-pgsql
```

Comment:
I disagree with changing this portion of the defaults file altogether.  There is a whole portion of this formula to deal with postgres instead of mysql.  Mysql is still valid, and there is no reason to change this in the defaults.